src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReaderOptions.cs

```diff
@@ -57,6 +57,16 @@ public ConfigurationReaderOptions() : this(dependencyContext: null)
     /// </summary>
     public IFormatProvider FormatProvider { get; init; } = CultureInfo.InvariantCulture;
 
+    /// <summary>
+    /// Allows to use internal types for extension methods for sink configuration. Defaults to <see langword="false"/>.
+    /// </summary>
+    public bool AllowInternalTypes { get; set; }
```

Comment:
I'd use **init** instead of **set**, like all other properties of this class. You probably don't want to change those properties after the options instance is created.

Reply:
ok